playwright.config.ts

```diff
@@ -22,7 +22,7 @@ export default defineConfig({
   /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
   use: {
     /* Base URL to use in actions like `await page.goto('/')`. */
-    baseURL: 'http://localhost:5045',
+      baseURL: 'http://localhost:5045',
```

Comment:
@SrushtiPasari94 should we update this to https endpoing?

Reply:
This also redirect to https.
Somehow with https playwright test are failing. So kept this as is and added middleware for httpsredirect.

Reply:
gotcha!